### ISSUE TITLE
PLEASE READ! If you are migrating from a different AppSales fork.

### ISSUE THREAD

**Issue by nicolasgomollon** [nicolasgomollon/AppSales-Mobile#1]
**1) Your payments will be duplicated after updating to this fork.**
    - To fix this, you will have to either: (a) manually update the vendor ID on the existing payments before fetching new ones, or (b) clear all of your payments before fetching new ones.
    - _NOTE:_ You will have to manually add in payments made prior to the past 24 months using some SQLite editor app. _(Detailed instructions will be added soon.)_
    - _WHY?_ This is due to a fix for a bug that’s been in the code for quite a long time (2d393d14e52180a3d03d3db68faf594d778c48ef), where the vendor ID was never saved with each payment, so those payments are now duplicated because they’re being re-saved with the proper vendor ID.

**2) You will run into issues where the app will crash when attempting to view previously downloaded reviews.**
    - To fix this, you will have to delete all previous reviews from the database and then download them again within the app. _(Detailed instructions will be added soon.)_
    - _NOTE:_ They should **all** be redownloaded again, since they’re now fetched directly from iTunes Connect.
    - _WHY?_ The database schema was changed significantly in order to support reviews grouped by version and new columns were added, so previous reviews will be missing these columns and relations, which will crash the app whenever accessed.

**Comment by pupasani** [comment]
For the 1st issue, will it work if I update the vendor ID on the existing payments manually first (using a SQL editor) before downloading new payments? Might be less work than having to re-input from scratch.

**Comment by nicolasgomollon** [comment]
@pupasani Yes, that’s definitely another option, so I’ve updated my post to include it. Thanks!

**Comment by xavrr** [comment]
Hi there, How do you import historical data ?  I only get a bit more than one month of data...

**Comment by ddaddy** [comment]
Use iTunes file sharing to copy your sales reports to the app, then use the import option in the app under our account settings.
